schematic/schemas/create_json_schema.py

```diff
@@ -206,19 +209,33 @@ def __post_init__(self) -> None:
             if type_rule:
                 self.type = TYPE_RULES.get(type_rule)
 
-            range_rule = _get_in_range_rule_from_rule_list(validation_rules)
+            regex_rule = _get_rule_from_rule_list("regex", validation_rules)
```

Comment:
All the changes look good and my only suggestion is for items like this where you have strings being used in the code. Usually I like to see these being string constants so that it remove the chance for a dev to mis-spell one of the checks somewhere.